src/event.rs

```diff
@@ -49,37 +49,37 @@ pub enum Event {
     SectionSplit(Prefix<XorName>),
     /// Our own section requires merged with others, resulting in the included `Prefix` for our new
     /// section.
-    SectionMerge(Prefix<XorName>),
+    SectionMerged(Prefix<XorName>),
     /// The client has successfully connected to a proxy node on the network.
     Connected,
     /// Disconnected or failed to connect - restart required.
     RestartRequired,
     /// Startup failed - terminate.
-    Terminate,
+    Terminated,
     // TODO: Find a better solution for periodic tasks.
     /// This event is sent periodically every time Routing sends the `Heartbeat` messages.
-    Tick,
+    Ticked,
```

Comment:
This name feels a bit weird :P I mean, it's not exactly the node that ticked, is it? Maybe `TimerTicked` or `TickHappened` or something like that?